internal/helpers/constructors.go

```diff
@@ -0,0 +1,38 @@
+package helpers
+
+import (
+	"encoding/json"
+	realopenslov1 "github.com/OpenSLO/OpenSLO/pkg/openslo/v1"
+	openslov1 "github.com/oskoperator/osko/api/openslo/v1"
+	oskov1alpha1 "github.com/oskoperator/osko/api/osko/v1alpha1"
+	"reflect"
+)
+
+func GetQuery(input realopenslov1.SLIMetricSource) string {
+	// return input.MetricSourceSpec["query"].(string)
+	// return string(input.MetricSourceSpec[])
+	// TODO: fix this
+
+	// test := []byte{}
+	// for _, v := range input.MetricSourceSpec {
+	// 	v.UnmarshalJSON(test)
+	// }
+	return "hello"
+}
```

Comment:
this will actually work similarly to the `ConstructConnectionDetails()` func. For now I just want the code to not error out :D

Reply:
this is done now as well